### ISSUE TITLE
Preload property for audio tag

### ISSUE THREAD

**Issue by avgustine** [vmudigal/ngx-audio-player#103]
It would be nice to have ability to change preload property of underlying audio tag(to "none" for example).
The real problem is when you need to have large number of players on the page and each of them trying to preload their playlists which resulting in a serious performance issues.

**Comment by nsnayp13** [comment]
Yeah. Really I thought this feature was a must-have for such project...

**Comment by Urinprobe** [comment]
Duplicate of #70
